tests/CMakeLists.txt

```diff
@@ -170,6 +171,7 @@ add_gtfilter(CxxEnum "CxxEnum*" True)
 add_gtfilter(CxxOpenClose "CxxOpen*" True)
 add_gtfilter(CxxBuffer "CxxEnum*" True)
```

Comment:
This might be a typo..  You want it to run the Enum tests when CxxBuffer is given?

Reply:
Oh! You're right! Fix coming up...